### PR TITLE
feat: didexchange interop between acapy and afgo

### PR DESCRIPTION
**Pull request by Moopli**
Signed-off-by: Filip Burlacu <filip.burlacu@securekey.com>

### CODE REVIEW

File: pkg/doc/did/serialize_default.go [hunk 1]
```diff
@@ -7,6 +7,9 @@ SPDX-License-Identifier: Apache-2.0
 
 package did
 
+// nolint:gochecknoglobals
+var doACAPYInterop = false
```
Comment (troyronda) [inline pkg/doc/did/serialize_default.go:11]: This should be a constant not a variable.

File: pkg/doc/did/serialize_interop.go [hunk 2]
```diff
@@ -17,6 +17,9 @@ import (
 	"github.com/btcsuite/btcutil/base58"
 )
 
+// nolint:gochecknoglobals
+var doACAPYInterop = true
```
Comment (troyronda) [inline pkg/doc/did/serialize_interop.go:21]: Constant not a variable.

**Review by fqutishat — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: pkg/doc/did/serialize_default.go (hunk 1 above)
Comment (Moopli) [reply pkg/doc/did/serialize_default.go]: fixed

File: pkg/doc/did/serialize_interop.go (hunk 2 above)
Comment (Moopli) [reply pkg/doc/did/serialize_interop.go]: fixed

**Review by troyronda — APPROVED** [review]

**Pull request merged**